cc.h

```diff
@@ -42,6 +42,14 @@
 // CONSTANT RISCV64 8
 #define RISCV64 8
 
+/* Stack grows to higher memory addresses */
+#define STACK_DIRECTION_PLUS 0
```

Comment:
Rest of the `defines` are after `CONSTANT` statement. Maybe let's keep  style consistent?

Reply:
Fixed.